src/mixins/view.js

```diff
@@ -163,11 +167,18 @@ export default {
     // https://github.com/jashkenas/backbone/blob/1.2.3/backbone.js#L1235
     this._removeElement();
 
+    if (shouldTriggerDetach) {
+      this._isAttached = false;
+      this.triggerMethod('detach', this);
+    }
+
     // remove children after the remove to prevent extra paints
     this._removeChildren();
 
     this._destroyBehaviors(args);
 
+    this._isDestroyed = true;
```

Comment:
related to this.. something I recently noticed.  view triggerMethods get proxied directly on the behavior like `onRender`... this works for everything now except for `onDestroy` `onDestroy` is called for the view and the behavior itself.  Granted what triggered it will be the view or the behavior as the 1st arg.. but it might make a case for knowing that the `onDestroy` of the behavior is occurring during the `onDestroy` of the view..  though I don't know the use case for destroying a behavior apart from the view